ai-data/llm-inference/reference-content/llama-2-7b-chat.mdx

```diff
@@ -49,34 +49,19 @@ To perform inference tasks with your Llama-2 deployed at Scaleway, use the follo
 
 ```bash
 curl -s \
--H "X-Auth-Token: <IAM API key>" \
+-H "Authorization: Bearer <IAM API key>" \
 -H "Content-Type: application/json" \
 --request POST \
---url "https://<Deployment UUID>.ifr.fr-par.scw.cloud/" \
---data '{"text_input": "[INST]There's a llama in my garden, what should I do? [/INST]", "max_tokens": 200, "temperature": 0.2, "random_seed": 1, "top_p": 0.9}' | jq -r .text_output
+--url "https://<Deployment UUID>.ifr.fr-par.scw.cloud/v1/chat/completions" \
+--data '{"messages":[{"role": "user","content": "There is a llama in my garden, what should I do?"}], "max_tokens": 200, "top_p": 1, "top_k": 40, "temperature": 1, "stream": false}'
 ```
 
 Make sure to replace `<IAM API key>` and `<Deployment UUID>` with your actual [IAM API key](/identity-and-access-management/iam/how-to/create-api-keys/) and the Deployment UUID you are targeting.
 
 <Message type="note">
-  Ensure that the text_input data is properly formatted according to the model's input requirements.
+  Ensure that the `messages` array is properly formatted with roles (system, user, assistant) and content.
 </Message>
 
-### Prompt engineering
-
-Here is an example with a format to define system and instruction prompts, designed as a virtual assistant to deliver only constructive and respectful responses. 
-
-```
-<s>[INST] <<SYS>>
-You are a helpful, respectful and honest assistant. Always answer as helpfully as possible, while being safe. Your answers should not include any harmful, unethical, racist, sexist, toxic, dangerous, or illegal content. Please ensure that your responses are socially unbiased and positive in nature.
```

Comment:
Should I transmit a system prompt like this in the openai-proxy, before the user messages? Or do we leave it to the user since they can provide messages with role "system"?

Reply:
This is for the user through system role :)